css/style.css

```diff
@@ -0,0 +1,101 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Montserrat', sans-serif;
+}
+body{
+    position: relative;
+    padding-bottom: 60px;
+}
+@media only screen and (max-width: 400px) {
+   div#products{
+      background-color: lightblue;
+      display: flex;
+      flex-direction: row;
+    }
+  }
+
```

Comment:
I think it would be better to put media queries at the end of your style sheet for better readability.